components/script/dom/htmliframeelement.rs

```diff
@@ -491,6 +490,18 @@ impl HTMLIFrameElementMethods for HTMLIFrameElement {
         element.set_bool_attribute(&local_name!("mozbrowser"), value);
     }
 
+    // https://html.spec.whatwg.org/multipage/#attr-iframe-allowfullscreen
+    fn AllowFullscreen(&self) -> bool {
+        let doc = document_from_node(self);
+        doc.get_allow_fullscreen()
+    }
+
+    // https://html.spec.whatwg.org/multipage/#attr-iframe-allowfullscreen
+    fn SetAllowFullscreen(&self, value: bool) {
+        let doc = document_from_node(self);
+        doc.set_allow_fullscreen(value);
```

Comment:
This won't work - this sets the allowed status for the document that contains the iframe, not the document contained inside the iframe.